### PR TITLE
Updated onboarding guide

### PR DESCRIPTION
**Pull request by armandomistral**
- Updated documentation to reflect the current implementation
- Step-by step instructions added for both 'Onboard as investor' and 'Onboard to pool'
- Removed section 'Why onboarding and KYC'

### CODE REVIEW

**Comment by denniswell** [comment]
@armandomistral You've deleted the required header
image
Without this the page cannot be built I think, that's probably also why all the checks are failing
image
All pages need to have this header, see here: https://centrifuge.hackmd.io/Z6hiOzK1Sea6yUAVOKa66Q#Structure-of-a-indexmd

**Comment by armandomistral** [comment]
Oops! You are right.
Yes, I added it back in.
(also created pull request for images)

File: docs/use/onboarding/index.md [hunk 1]
```diff
@@ -2,125 +2,175 @@
...
 
-This obviously contradicts the idea of decentralized and open finance and the Centrifuge team is working hard to allow more people to invest within the legal framework given by regulators.
+**Onboarding to a pool** requires:
+- Ethereum wallet
+- A verified KYC in Securitize (i.e. a completed onboarding as an investor)
+- A minimum investment amount of 5,000 DAI
 
-## Onboarding quick guide
+#### Further requirements
+- **US investors** need to be "Acreddited investors" to invest in Tinlake pools. *See section [Onboarding as US investor](#Onboarding-as-an-US-investor)*
```
Comment (denniswell) [inline docs/use/onboarding/index.md:30]: @onnovisser Could you help us out and double check that this is the correct syntax for referencing to a section within this document? Not working properly for me on the preview

Comment (onnovisser) [reply docs/use/onboarding/index.md:30]: is this used anywhere else? some features of markdown might not be supported out-of-the-box. we might need to add a plugin to make this work

**Review by denniswell — APPROVED** [review]

**Pull request merged**